contracts/zero-ex/contracts/src/transformers/bridges/IBridgeAdapter.sol

```diff
@@ -18,12 +18,51 @@
 */
 
 pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+
 
 interface IBridgeAdapter {
 
+    struct BridgeOrder {
+        uint256 source;
+        uint256 takerTokenAmount;
+        uint256 makerTokenAmount;
+        bytes bridgeData;
+    }
+
+    struct Addresses {
```

Comment:
should we just force these into `bridgeData`? Apart from WETH

Reply:
I've thought about it. I wasn't sure if it was a good idea to change the bridge data encoding too much.

Reply:
Guess it does make FQT overrides in simbot easier?

Reply:
I did it